test/sslapitest.c

```diff
@@ -1911,6 +1911,9 @@ static int test_tlsext_status_type(void)
     /* First just do various checks getting and setting tlsext_status_type */
 
     clientssl = SSL_new(cctx);
+    if (clientssl == NULL)
+        goto end;
+        
```

Comment:
This needs to be changed to `if (!TEST_ptr(clientssl))` and similarly below.

Reply:
OK, I'll fix it